packages/eas-cli/src/analytics.ts

```diff
@@ -96,6 +96,7 @@ function getContext() {
     os_version: os.release(),
     device_brand: platform,
     device_model: platform,
-    app_version: packageJSON?.version,
+    source_version: packageJSON?.version,
```

Comment:
the action stat is using the convention of `source` & `source_version`. 

I think it would be good to use this convention for unified events in general as the project gathers disparate events into one place.

That said, I am ok with leaving the context alone and putting this into the log event call itself to avoid changing what's already here